src/v2/api/controllers/UserController.ts

```diff
@@ -898,51 +898,127 @@ export class UserController {
     return this.userMapper.getUser(user);
   }
 
-  @Access(PERMISSION.USERS_$USERID_SELECTIVE_GET)
-  @Get('/:userId/selectiveDisciplines')
+  @ApiBearerAuth()
   @ApiOkResponse({
     type: RemainingSelectiveResponse,
   })
-  @ApiBadRequestResponse({
-    description: `InvalidEntityIdException:\n 
-                  User with such id is not found`,
+  @ApiUnauthorizedResponse({
+    description: `\n
+    UnauthorizedException:
+      Unauthorized`,
   })
   @ApiForbiddenResponse({
-    description: `NoPermissionException:\n
-                  You do not have permission to perform this action`,
+    description: `\n 
+    NoPermissionException:
+      You do not have permission to perform this action`,
+  })
+  @ApiBadRequestResponse({
+    description: `\n
+    InvalidEntityIdException: 
+      User with such id is not found
+    DataNotFoundException: 
+      Data were not found`,
   })
+  @ApiEndpoint({
+    summary: 'Get all user\'s selective disciplines',
+    permissions: PERMISSION.USERS_$USERID_SELECTIVE_GET,
+  })
+  @Get('/:userId/selectiveDisciplines')
   async getRemainingSelective (
     @Param('userId', UserByIdPipe) userId: string,
     @Query() body: RemainingSelectiveDTO,
```

Comment:
if you choose to describe the documentation for this endpoint, finish it by adding descriptions to the body and response

Reply:
💩💩💩